share/xslt/positive-config.xsl

```diff
@@ -51,7 +51,7 @@
 
 
   <xsl:template match="deliverable" mode="create-blacklist-param">
-    <xsl:param name="dc" select="dc"/>
+    <xsl:param name="DC" select="dc"/>
```

Comment:
> I don't really get oxygen's complaint

Well, if I'm not mistaken, I guess it's probably to avoid any ambiguities. It's easy to forget a `$` symbol on a name. However, if your variables are well named (and don't conflict with element names), then it avoids any issues. I had the same issue once, so I think, it's probably a good style.

Actually, that commit shouldn't be there in this branch. For some reason, I probably created and branched it. I will deal with that issue in master.

Reply:
Please just fix the variable names here now that you've touched them in this branch anyway.

Reply:
That's done in commit e683eb8

Reply:
I had hoped you'd agreed with the suggestion `$current-dc`/`current-subdeliverable` ...

Reply:
Uups, then it was slipped through. Will change it accordingly...

Reply:
Ok, this is now done in commit 906c1054e.
I realized, that the dash between `current` and `dc` can be problematic so I avoided it. I didn't find any `$subdeliverable` variable.

Reply:
Seems it's not my day.. sigh. Another fix, this time I hope I caught all variables in commit e7848cd.